app/controllers/provider_interface/decisions_controller.rb

```diff
@@ -14,7 +14,36 @@ def submit_response
     end
 
     def new_offer
-      raise 'Not yet implemented'
+      @application_offer = MakeAnOffer.new(application_choice: @application_choice)
+    end
+
+    def confirm_offer
+      standard_conditions_array = params.dig(:make_an_offer, :standard_conditions)
+      further_conditions_array = [params.dig(:make_an_offer, :first_condition), params.dig(:make_an_offer, :second_condition), params.dig(:make_an_offer, :third_condition), params.dig(:make_an_offer, :further_condition)].reject(&:blank?)
```

Comment:
Did you consider having a single "further_conditions" array which is numerically indexed? What's the downside there? We could do this:

which will reduce `dig`ging up here:

```
offer_conditions = [
  make_an_offer_params[:standard_conditions],
  make_an_offer_params[:further_conditions]
].flatten

...

def make_an_offer_params
  params.require(:make_an_offer)
end
```